features/support/content_register.rb

```diff
@@ -0,0 +1,7 @@
+require_relative '../../test/support/content_register_helpers'
+
+Before do
```

Comment:
Can we do this in a cucumber tag?  We don't need these stubbed on every feature.